### DISCUSSION TITLE
How can I make the text to rotate ?

### DISCUSSION THREAD

**Discussion by vladQll** [xournalpp/xournalpp#4614] [Q&A]
I have noticed that the text cannot be rotated, I would like to know how to rotate it so that it can be placed near a segment, or vector so that it occupies less space on the graph because horizontally when I have a lot of data, magnitudes, angles, it looks very saturated.
![image](https://user-images.githubusercontent.com/28522011/213935939-4a71a6a6-b2b3-4ca5-982f-ec76df4ee772.png)
por ejemplo el 100N me gustaria rotarlo para colocarlo diagonalmente junto al vector azul

**Comment by rolandlo** [comment]
You can use LaTeX for rotated text, see [website](https://xournalpp.github.io/guide/tools/latex/#template). The text tool currently doesn't support it.
